lib/quby/questionnaires/dsl/charting/chart_builder.rb

```diff
@@ -31,6 +31,10 @@ def tick_interval(tick_interval)
           @chart.tick_interval = tick_interval
         end
 
+        def y_categories(y_categories)
```

Comment:
Ik had em per ongeluk gemerged met master ipv gerebased, foutje. Maar alle wijzigingen die ondertussen in master zijn gedaan, zitten dus al wel in deze branch.